incubator/airflow/templates/NOTES.txt

```diff
@@ -0,0 +1,28 @@
+Congratulations. You have just deployed Apache Airflow
+
+1. Get the Artifactory URL by running these commands:
+
+   {{- if .Values.ingress.enabled }}
+    - Web UI: http://{{ .Values.ingress.web.host }}{{ .Values.ingress.web.path }}
+    - Flower: http://{{ .Values.ingress.flower.host }}{{ .Values.ingress.flower.path }}
+
+   {{- else if contains "NodePort" .Values.airflow.service.type }}
+   export NODE_PORT=$(kubectl get --namespace {{ .Release.Namespace }} -o jsonpath="{.spec.ports[0].nodePort}" services {{ template "airflow.fullname" . }})
+   export NODE_IP=$(kubectl get nodes --namespace {{ .Release.Namespace }} -o jsonpath="{.items[0].status.addresses[0].address}")
+   echo http://$NODE_IP:$NODE_PORT/
+
+   {{- else if contains "LoadBalancer"  .Values.airflow.service.type }}
+
+   NOTE: It may take a few minutes for the LoadBalancer IP to be available.
+         You can watch the status of the service by running 'kubectl get svc -w {{ template "airflow.fullname" . }}'
+   export SERVICE_IP=$(kubectl get svc --namespace {{ .Release.Namespace }} {{ template "airflow.fullname" . }} -o jsonpath='{.status.loadBalancer.ingress[0].ip}')
+   echo http://$SERVICE_IP/
+
+   {{- else if contains "ClusterIP" .Values.airflow.service.type }}
+   export POD_NAME=$(kubectl get pods --namespace {{ .Release.Namespace }} -l "component={{ .Values.airflow.name }}" -o jsonpath="{.items[0].metadata.name}")
```

Comment:
I am getting the following output when I install the chart:

```   
export POD_NAME=$(kubectl get pods --namespace default -l "component=" -o jsonpath="{.items[0].metadata.name}")
echo http://127.0.0.1:
kubectl port-forward --namespace default $POD_NAME :
```

When I run the first line, I get an error:

```
error: error executing jsonpath "{.items[0].metadata.name}": array index out of bounds: index 0, length 0
```

Reply:
not sure how to help, we use ClusterIP and it works